sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -41,7 +39,17 @@ export class IdentityClient extends ServiceClient {
 
   constructor(options?: IdentityClientOptions) {
     options = options || IdentityClient.getDefaultOptions();
-    super(undefined, options);
+
+    const internalPipelineOptions = {
+      ...options,
+      ...{
+        loggingOptions: {
+          logger: logger.info,
```

Comment:
Does this mean that the pipeline logging will log under the `azure:identity` namespace?

Reply:
Yep, is that undesirable?  If so, I can remove it.

Reply:
When it's not set, the logs show up as being from `core-http`.

Reply:
I thiiiiink core-http is better, but I'm not sure... I'm imagining people enabling azure:identity to get basic semantic info but don't want to see raw requests/responses so don't turn on core-http logging?

Reply:
Currently there's not a way in `PipelineOptions` to say you don't want a `logPolicy`; not providing a `logger` here will just default to `core-http`'s logger.  Should I add an `enable` field for that in LoggingOptions?

Reply:
Is it enough to rely entirely on AZURE_LOG_LEVEL/DEBUG env vars for enabling/disabling?

Reply:
For overall logging enable/disable, yeah, but to get rid of `core-http` log messages from within the IdentityClient we'd need to remove the `logPolicy` from the pipeline that `createPipelineFromOptions` creates.

Maybe I'm misunderstanding and you're saying that by letting the requests get logged through `core-http`, the user can selectively turn on `info` logging for _only_ `identity`?

Reply:
Isn't always having a logPolicy fine? It shouldn't log unless logging is enabled.

Also yes, if you enable just identity logging (either by calling `enable` on the logger directly, or by setting DEBUG to `azure:identity:*`, you wouldn't get raw requests/responses from core. I'm not sure if this is desirable or not, fwiw, just observing that it's possible.

Reply:
Cool, so I'll just take out the `logger` assignment here and let `core-http`'s logger be used in the `logPolicy` for `IdentityClient`'s pipeline.